provisioning/device/src/ProvisioningDeviceClient.cs

```diff
@@ -110,14 +103,11 @@ public Task<DeviceRegistrationResult> RegisterAsync(CancellationToken cancellati
         /// <returns>The registration result.</returns>
         public Task<DeviceRegistrationResult> RegisterAsync(ProvisioningRegistrationAdditionalData data, CancellationToken cancellationToken)
         {
-            if (Logging.IsEnabled)
-            {
-                Logging.RegisterAsync(this, _globalDeviceEndpoint, _idScope, _transport, _security);
-            }
+            Logging.RegisterAsync(this, _globalDeviceEndpoint, _idScope, _transport, _security);
 
             var request = new ProvisioningTransportRegisterMessage(_globalDeviceEndpoint, _idScope, _security, data?.JsonData)
             {
-                ProductInfo = ProductInfo
+                ProductInfo = ProductInfo,
```

Comment:
Nit: the reason I add `,` (and why it's supported in the language) in a list is that a future addition doesn't touch the line above (and the diff is one line smaller).

Reply:
That's why I add it too. I don't understand your nit, though if you like having it, and I added it.